bin/test-server.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const firefoxDriver = require("./firefox-driver");
+
+const By = firefoxDriver.By;
+const until = firefoxDriver.until;
+const Key = firefoxDriver.Key;
+
+let driver;
+let isQuiting = false;
+let isOpen = false;
+
+function debuggeeCommand(req, res) {
+  const command = req.body.command;
+  const timeout = req.body.timeout;
+  const formattedCommand = command.split("\n").map(s=> "  " + s.trim()).join("\n")
+  console.log("Initiating command\n" + formattedCommand)
+
+  var isDone = false;
+  var cb = () => {
+    if (!isDone) {
+      console.log("Command Executed\n");
+      isDone = true;
+      if (timer) {
+        clearTimeout(timer);
+      }
+      res.sendStatus(200);
+    }
+  }
+
+  try {
+    var r  = eval(command);
+  } catch (e) {
+    console.log("error", e.stack);
+  }
+
+  var timer = setTimeout(cb, timeout);
+  r.then(cb)
```

Comment:
Not sure I understand this; the `eval` happens right above this so the timer isn't installed before the `eval` finishes executing. But this is all magic to me.